### PR TITLE
nyse-example Updated nyse example to perform per day aggregations

### PR DESCRIPTION
**Pull request by vigbk**
…ng the newly generated data

#### Pull Request Checklist:
- [x] Tests for the changes have been added
- [x] Docs have been added / updated

### What kind of change does this PR introduce?
_Describe the big picture of your changes here_

#### Link to issue
fixes #issue-number

### CODE REVIEW

File: docs/examples/nyse/nyse.ipynb [hunk 1]
```diff
@@ -20,7 +20,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 44,
```
Comment (prashastk) [inline docs/examples/nyse/nyse.ipynb]: Clean up the execution counts (start from 1?)

File: docs/examples/nyse/nyse.ipynb [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "import sys\n",
     "\n",
     "print(\"installing blurr, keras and tensorflow...\")\n",
```
Comment (prashastk) [inline docs/examples/nyse/nyse.ipynb]: Install pandas as well. Its being used lower below.

File: docs/examples/nyse/nyse-streaming-dtc.yml [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
+Time: parse(source.datetime)
 
 Stores:
-  - Type: Blurr:Store:Memory
+  - Type: 'Blurr:Store:Memory'
     Name: memory
 
 Aggregates:
-  - Type: Blurr:Aggregate:Block
+  - Type: 'Blurr:Aggregate:Block'
```
Comment (prashastk) [inline docs/examples/nyse/nyse-streaming-dtc.yml]: No need to have the types within quotes. Here and above.

File: docs/examples/nyse/nyse-streaming-dtc.yml [hunk 4]
```diff
@@ -1,34 +1,49 @@
...
-        Value: (float(source.high) / float(source.low)) - 1
+        Value: (float(stats.high) / float(stats.low)) - 1
+        When: stats.low > 0
 
       - Name: volume
         Type: float
-        Value: source.volume
+        Value: stats.volume + source.volume
+
+      - Name: latest_tradetime
```
Comment (prashastk) [inline docs/examples/nyse/nyse-streaming-dtc.yml]: why not store the date directly instead of latest_tradetime?

Comment (vigbk) [reply docs/examples/nyse/nyse-streaming-dtc.yml]: we don't support the date type. We only support datetime.

File: docs/examples/nyse/nyse-streaming-dtc.yml (hunk 3 above)
Comment (vigbk) [reply docs/examples/nyse/nyse-streaming-dtc.yml]: Noted.

File: docs/examples/nyse/nyse-streaming-dtc.yml (hunk 4 above)
Comment (prashastk) [reply docs/examples/nyse/nyse-streaming-dtc.yml]: i simplified this by using `str(time.date()) != stats.date` as the split condition

**Pull request merged**